CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -1078,6 +1151,13 @@ public CommandBase Delete
                             {
                                 throw new NotImplementedException("Not implemented");
                             }
+                            else if (itemType == typeof(CertificateModel))
+                            {
+                                CertificateModel certificate = item as CertificateModel;
+
+                                objectType = "Certificate";
+                                objectName = certificate.Thumbprint;
```

Comment:
Shouldn't objectName include both of the primary keys of the certificate (Thumbprint and Algorithm I think?)  This is a message to the user so you can just do like string.Format("Thumbprint={0}, Algorithm={1}", ...) or whatever is clearest.